plexupdate.sh

```diff
@@ -307,6 +307,20 @@ if [ -z "${DISTRO_INSTALL}" ]; then
 			else
 				DISTRO_INSTALL="${REDHAT_INSTALL}"
 			fi
+		elif [ -f /etc/synoinfo.conf ]; then
+		        DISTRO="synology"
+		        if [ "${PUBLIC}" = "yes" ]; then
+			        if [ "${ARCH}" = "x86_64" ]; then
```

Comment:
I believe these build values are no longer accurate. This is what I see when I pull down the builds from Plex.

        "Synology": {
            "id": "synology",
            "name": "Synology",
            "release_date": 1557930938,
            "version": "1.15.6.1079-78232c603",
            "requirements": "Note: <b>Requires DSM 5.0 minimum to install.</b>",
            "extra_info": "Guide: <a href=\"https://support.plex.tv/hc/en-us/articles/205165858\" target=\"_blank\">Add Plex as a trusted publisher for package installations</a>.",
            "items_added": "(DVR) Add `Resolution`, `Replace lower resolution items`, and `Allow partial airings` at DVR level with option to override at recording level.",
            "items_fixed": "(Analysis) Corrected chapters not being imported again\r\n(DVR) Prefer HD airings when both HD and SD are available. (#9641)\r\nDocker host addresses were published unnecessarily (#9576)\r\nIP addresses from interfacing starting with a \"v\" weren'\''t published (#9576)\r\nSync or Optimize could fail to fallback to software transcoding if hardware transcoding failed (#9455)",
            "releases": [{
                "label": "Intel 32-bit (x10 Series, DS415play, and DS214play)",
                "build": "linux-x86",
                "distro": "synology",
                "url": "https://downloads.plex.tv/plex-media-server-new/1.15.6.1079-78232c603/synology/PlexMediaServer-1.15.6.1079-78232c603-x86.spk",
                "checksum": "1a6e5fceaf23031dab8518929c3dd8c2597fd93b"
            }, {
                "label": "Intel 64-bit (DSM 6.0 and newer)",
                "build": "linux-x86_64",
                "distro": "synology",
                "url": "https://downloads.plex.tv/plex-media-server-new/1.15.6.1079-78232c603/synology/PlexMediaServer-1.15.6.1079-78232c603-x86_64.spk",
                "checksum": "b1fb36ddce473b7518248e6ef67c1d01500fb72c"
            }, {
                "label": "ARMv8 (x18 Series)",
                "build": "linux-aarch64",
                "distro": "synology",
                "url": "https://downloads.plex.tv/plex-media-server-new/1.15.6.1079-78232c603/synology/PlexMediaServer-1.15.6.1079-78232c603-aarch64.spk",
                "checksum": "efc4caa3adc94347705b9e8483f0fd23c774d493"
            }, {
                "label": "ARMv7 (x13 Series, x14 Series (excluding DS4v14j), DS115j, RS815, and DS216se)",
                "build": "linux-armv7hf",
                "distro": "synology",
                "url": "https://downloads.plex.tv/plex-media-server-new/1.15.6.1079-78232c603/synology/PlexMediaServer-1.15.6.1079-78232c603-armv7hf.spk",
                "checksum": "7845bced023c11beb54e6a2b4cccf2bdbc20f55a"
            }, {
                "label": "ARMv7 (x15 Series (excluding DS115j and RS815), x16 Series (excluding DS216se), x17 Series, x18 Series, and DS414j)",
                "build": "linux-armv7hf_neon",
                "distro": "synology",
                "url": "https://downloads.plex.tv/plex-media-server-new/1.15.6.1079-78232c603/synology/PlexMediaServer-1.15.6.1079-78232c603-armv7hf_neon.spk",
                "checksum": "f3b85a650ce6c2bb7b5fe654e1758cf9c2877963"
            }]
        },

Reply:
Is that for public or private builds? When I added that stuff, only plexpass builds were using the new build values. Which is why if ${PUBLIC} is no, it uses
`else
				BUILD="linux-${ARCH}"`

If non-plexpass build values are now updated, I can remove those lines

Reply:
This was for public builds.

Reply:
just fixed it